cogs/welcome/welcome.py

```diff
@@ -132,6 +199,139 @@ async def logServerLeave(self, leaveUser: discord.Member):
     async def welcome(self, ctx: Context):
         """Server welcome message settings."""
 
+    @checks.mod_or_permissions()
+    @welcome.command(name="channel")
+    async def welcomeChannelSet(self, ctx: Context, channel: discord.TextChannel):
```

Comment:
Looks like we can't "unset" after we've set a channel. Perhaps we can we set a default here?
```suggestion
    async def welcomeChannelSet(self, ctx: Context, channel: discord.TextChannel = None):
```

This way, inside this method, you can do a check for `None`, which would deconfigure this, and we would no longer post to a welcome channel.

Reply:
done